deploy-disconnected-registry/olm-sync.sh

```diff
@@ -42,8 +42,14 @@ function prepare_env() {
 function check_registry() {
     REG=${1}
 
+
+
     for a in {1..30}; do
-        skopeo login ${REG} --authfile=${PULL_SECRET} --username ${REG_US} --password ${REG_PASS}
+        if [[ ${CUSTOM_REGISTRY} == "false" ]]; then
+            skopeo login ${REG} --authfile=${PULL_SECRET} --username ${REG_US} --password ${REG_PASS}
+        else
+            skopeo login ${REG} --authfile=${PULL_SECRET}
```

Comment:
you don't need credentials right? are included into the authfile, aren't they?

Reply:
yes that's was agreed with @flaper87  - 

on that note - you must update the openshift PS with the right domain and creds 

`oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=updated-pull-secret.json`